es6-template-literals/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Javascript ES6 - Template Literals</title>
+  </head>
+  <body>
+    <script>
+      const number = 9200;
+      const street = 'Irvine Center Drive';
+
+      console.log(`LearningFuze: ${number} ${street}`);
+
+      const firstName = 'Uzair';
+      const lastName = 'Ashraf';
+      const age = 25;
+
+      const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+      console.log(`Bio: ${bio}`);
+
+      const fruit = 'apple';
+      const item = 'pen';
+
```

Comment:
There should be a variable here `applePen` where the template literal gets created